spec/post_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:title) }
+
+    it 'should validate that comments_counter is greater than or equal to 0' do
+      post = Post.new(title: 'title', comments_counter: -1, likes_counter: 1)
+      expect(post).not_to be_valid
+      expect(post.errors[:comments_counter]).to include('must be greater than or equal 0')
+    end
+
+    it 'should validate that likes_counter is greater than or equal to 0' do
+      post = Post.new(title: 'title', comments_counter: 1, likes_counter: -1)
+      expect(post).not_to be_valid
+      expect(post.errors[:likes_counter]).to include('must be greater than or equal 0')
+    end
```

Comment:
Writing the tests this way will always produce an error when the correct inputs are given. For instance, when you put in a comment counter or likes counter to be equal to a postive number greater than 0, it will always produce an error as shown below. Kindly fix it to ensure that the all test pass when the comment/likes counter is greater than or equal to 0, and all tests fail when the comment/likes counter is less than 0.

![Screenshot from 2023-09-30 19-24-42](https://github.com/yin-ka/blog-app/assets/65233860/ac8c82e7-6761-4062-8049-a193a30aaff0)